src/Initializer/TaxCodeInitializer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Aligent\FixturesBundle\Initializer;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\Persistence\ObjectManager;
+use Oro\Bundle\TaxBundle\Entity\CustomerTaxCode;
+
+class TaxCodeInitializer implements ReferenceInitializerInterface
```

Comment:
Should this be `CustomerTaxCodeInitializer` so we can also have a `ProductTaxCodeInitializer` ?

Reply:
Good call I'll make a note of that and update shortly!